### PR TITLE
[CHK-2649] misc: update and customize java template

### PR DESCRIPTION
**Pull request by giovanniberti**
#### List of Changes

 * update and customize java template

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by petretiandrea — APPROVED** [review]
🚀

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,36 +0,0 @@
-<!--- Please always add a PR description as if nobody knows anything about the context these changes come from. -->
```
Comment (pietro-tota) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: why removing pull request template?

File: helm/values-dev.yaml [hunk 2]
```diff
@@ -29,8 +27,8 @@ microservice-chart:
       - 8080
   ingress:
     create: true
-    host: "your.host" # TODO: set the host
-    path: /your-path-here/(.*) # TODO: set your path
+    host: "weudev.wallet.internal.dev.platform.pagopa.it"
+    path: /pagopa-wallet-event-dispatcher-service/(.*) # TODO: set your path
```
Comment (pietro-tota) [inline helm/values-dev.yaml:31]:
```suggestion
    path: /pagopa-wallet-event-dispatcher-service/(.*)
```

File: README.md [hunk 3]
```diff
@@ -35,13 +40,28 @@ See the [OpenApi 3 here.](https://editor.swagger.io/?url=https://raw.githubuserc
...
+```shell
+$ cp .env.example .env
+```
 
-ℹ️ Note: for PagoPa ACR is required the login `az acr login -n <acr-name>`
+to get a good default configuration.
+
+If you want to customize the application environment, reference this table:
+
+| Variable name                                            | Description                                                        | type                     | default |
```
Comment (pietro-tota) [inline README.md:57]: add env variables here

**Review by pietro-tota — APPROVED** [review]
well done, just some considerations

File: README.md (hunk 3 above)
Comment (giovanniberti) [reply README.md:57]: For now there are no env variables :)

File: helm/values-dev.yaml (hunk 2 above)
Comment (giovanniberti) [reply helm/values-dev.yaml:31]: Fixing

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (giovanniberti) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: Readded in dfee437

File: gradle/libs.versions.toml [hunk 4]
```diff
@@ -0,0 +1,42 @@
+# This file was generated by the Gradle 'init' task.
+# https://docs.gradle.org/current/userguide/platforms.html#sub::toml-dependencies-format
+
+[versions]
```
[the remark this answers is not in the document]
Comment (giovanniberti) [reply gradle/libs.versions.toml]: Removed in e104998

**Pull request merged**